gaTable.yaml

```diff
@@ -994,3 +994,21 @@
   projectContactEmail: 'lplatt@usgs.gov'
   analyticsContact: 'David Watkins'
   analyticsContactEmail: 'wwatkins@usgs.gov'
+  
+- longName: National Water Dashboard
```

Comment:
needs quotes?

Reply:
`read_yaml` seems to be smart enough that doesn't matter.  After checking that manually though, a good easy test to implement would be to make sure this file reads in without errors 🤔